processor/spanmetricsprocessor/processor.go

```diff
@@ -30,6 +30,8 @@ import (
 	"go.opentelemetry.io/collector/model/pdata"
 	conventions "go.opentelemetry.io/collector/model/semconv/v1.5.0"
 	"go.uber.org/zap"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/processor/spanmetricsprocessor/cache"
```

Comment:
Why has it is included itself within the `go.mod`, not sure that is expected?

Reply:
I don't find it in `go.mod`

Reply:
Sorry, I miss read the file name 🤦🏽